src/domain/page/Page.tsx

```diff
@@ -22,12 +23,30 @@ const Page: React.SFC = ({ children }) => {
               key="harbors"
               onClick={() => history.push('/harbors')}
               label={
-                <InternalNavLink
-                  to="harbors"
-                  icon={<Icon name="fence" color="standard" />}
-                >
-                  <Text bold>Venepaikat</Text>
-                </InternalNavLink>
+                <InternalLink to="harbors">
+                  <Button
+                    variant="text"
+                    icon={<Icon name="fence" color="standard" />}
+                    onClick={() => {}}
+                  >
+                    <Text bold>Venepaikat</Text>
+                  </Button>
+                </InternalLink>
```

Comment:
I do prefer giving the button an `onClick` prop that takes care of the navigation like: `() => history.push('harbors');`
rather than wrapping the whole button by a link and giving the button a dummy function.

Reply:
HarborsContainer handles navigation without history so I decided to remove it for now. I also removed the dummy functions since they were redundant.